wazuh-indexer/config/config.sh

```diff
@@ -22,7 +22,7 @@ export REPO_DIR=/unattended_installer
 
 rm -rf ${INSTALLATION_DIR}/
 
-curl -o ${INDEXER_FILE} https://packages-dev.wazuh.com/stack/indexer/base/${BASE_FILE}
+curl -o ${INDEXER_FILE} https://packages.wazuh.com/stack/indexer/base/${BASE_FILE}
```

Comment:
The same for this Base URL: 
>The base is always downloaded from packages-dev. The base is an artifact to create dev OR prod artifacts but is always a dev artifact.

Reply:
Done in https://github.com/wazuh/wazuh-docker/pull/593/commits/862b35881cd7ba88ee8d01054d63467599bfb8ba